docs/projects/insertion-sort/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Insertion Sort in Every Language 
+layout: default
+date: 2018-12-16
+last-modified: 2018-12-16
+featured-image:
+tags: [insertion-sort]
+authors:
+  - auroq
+---
+
+In this article, we'll tackle Insertion Sort, an inefficient sorting algorithm.
+
+## Description
+
+Insertion sort is an algorithm that generally operates on a single list in place. It tracks a pointer
+that iterates through the list a single time, takes each item and inserts it sorted at the beginning
+of the list. By the time the pointer has iterated through the entire list, the beginning or
+now the entire list is now sorted.
```

Comment:
Redundant use of "now".

Reply:
Actually, I'm confused what this sentence is saying (granted it is late).

Reply:
I didn't like that either, but I wasn't sure how to fix it earlier. I think I got it now.